### PR TITLE
QT1-322 view apprenticeship employer and provider sides 

### PR DESCRIPTION
**Pull request by Pauljgraham**

### CODE REVIEW

**Review by alireid — DISMISSED** [review]
Working great @Pauljgraham , hooked it up to my local ProviderCommitments to run a full work flow.

File: src/Approvals/SFA.DAS.Approvals.Api/Controllers/DraftApprenticeshipController.cs [hunk 1]
```diff
@@ -75,6 +76,30 @@ public async Task<IActionResult> GetEditDraftApprenticeship(long cohortId, long
...
+            try
+            {
+                var result = await _mediator.Send(new GetViewDraftApprenticeshipQuery { CohortId = cohortId, DraftApprenticeshipId = draftApprenticeshipId });
+
+                if (result == null)
+                {
+                    return NotFound();
+                }
+
+                return Ok(result);
```
Comment (chrisfoster76) [inline src/Approvals/SFA.DAS.Approvals.Api/Controllers/DraftApprenticeshipController.cs:93]: We're not mapping to a response here and just returning the query result.

**Review by chrisfoster76 — CHANGES_REQUESTED** [review]

File: src/Approvals/SFA.DAS.Approvals.Api/Controllers/DraftApprenticeshipController.cs (hunk 1 above)
Comment (Pauljgraham) [reply src/Approvals/SFA.DAS.Approvals.Api/Controllers/DraftApprenticeshipController.cs:93]: ok, will map it thx

Comment (Pauljgraham) [reply src/Approvals/SFA.DAS.Approvals.Api/Controllers/DraftApprenticeshipController.cs:93]: fixed

**Review by chrisfoster76 — APPROVED** [review]

**Review by alireid — APPROVED** [review]

**Pull request merged**